src/test/resources/org/wildfly/test/integration/vdx/transformations/messaging/InvalidAddressSettingFullPolicy.groovy

```diff
@@ -0,0 +1,15 @@
+attr = [name: '#']
+attr['dead-letter-address'] = 'jms.queue.DLQ'
+attr['expiry-address'] = 'jms.queue.ExpiryQueue'
+attr['max-size-bytes'] = '10485760'
```

Comment:
max-size-bytes could be parametrized, it was used in different groovy script too - for too long value